docs/guides/cloud-setup.mdx

```diff
@@ -49,21 +49,26 @@ Your user account is associated with one or more [instances](./instances) that g
 
 1. Find your access credentials.
     1. Find your API key. From the [dashboard](https://quantum.cloud.ibm.com/), create your API key, then copy it to a secure location so you can use it for authentication.
-    2. Find your CRN from the Instances section of the dashboard. Click the icon to copy your CRN for the instance you want to use, then save it in a secure location so you can use it for authentication.
+    2. Optional: Find your CRN from the Instances section of the dashboard. Click the icon to copy your CRN for the instance you want to use, then save it in a secure location so you can use it to identify the instance.
 
 1. <span id="cloud-save"></span>**If you are working in a trusted Python environment (such as on a personal laptop or workstation),** use the `save_account()` method to save your credentials locally. ([Skip to the next step](#cloud-untrusted) if you are not using a trusted environment, such as a shared or public computer, to authenticate to IBM Cloud.) To use `save_account()`, activate a Python virtual environment. Then, run Python in your virtual environment and enter the following:
 
 
     ```python
     token = "<your-API-token>"
     ```
+<Admonition type="Note">
+When an instance CRN or name is passed in, only backends and jobs from that instance are available.  If an instance is not specified, all backends and jobs across all instances in your account are available. In this case, when a backend is specified, an instance that can access that backend is found and used.
+</Admonition>
```

Comment:
I think it'd be good to mention somewhere that not specifying an instance only works in `qiskit-ibm-runtime > 0.40.1`